packages/orbit-components/src/LinkList/index.tsx

```diff
@@ -99,24 +15,34 @@ const LinkList = ({
   children,
   dataTest,
 }: Props) => (
-  <StyledLinkList
-    $indent={indent}
-    $direction={direction}
+  <ul
     data-test={dataTest}
-    $legacy={legacy}
-    $spacing={spacing}
+    className={cx(
+      "flex",
+      "w-full",
+      "m-0 p-0",
+      "list-none",
+      "text-normal",
+      indent && "ps-xxs",
+      direction && getDirectionClasses(direction),
+      spacing && getSpacingClasses(spacing, undefined, direction, legacy),
+    )}
   >
     {React.Children.map(children, item => {
-      if (React.isValidElement(item)) {
-        return (
-          <StyledNavigationLinkListChild $direction={direction} $spacing={spacing} $legacy={legacy}>
-            {item}
-          </StyledNavigationLinkListChild>
-        );
-      }
-      return null;
+      if (!React.isValidElement(item)) return null;
+      return (
+        <li
+          className={cx(
```

Comment:
Is the `resolveSpacings` taken into account here?

Reply:
Not needed, I checked with the legacy flag and it worked fine. With Tailwind is enough to have it on the wrapper, as it targets children and adds margin 🤔